packages/fluxible-router/lib/navigateAction.js

```diff
@@ -12,7 +12,7 @@ function navigateAction (context, payload, done) {
         transactionId: context.rootId
     }, payload);
     if (!payload.url && payload.routeName) {
-        navigate.url = routeStore.makePath(payload.routeName, payload.params);
+        navigate.url = routeStore.makePath(payload.routeName, payload.params, payload.query);
```

Comment:
This is actually more important than the other file changes. makePath now supports query and only makes sense to accept query params on navigateAction payload.